system/jlib/jlog.cpp

```diff
@@ -2511,17 +2588,18 @@ MODULE_EXIT()
         theReporters[compo] = NULL;
     }
     delete theManager;
+    theManager = &nullManager;
     delete theSysLogEventLogger;
     delete theStderrHandler;
     delete thePassNoneFilter;
     delete thePassLocalFilter;
     delete thePassAllFilter;
-    theManager = NULL;
-    theSysLogEventLogger = NULL;
-    theStderrHandler = NULL;
-    thePassNoneFilter = NULL;
-    thePassLocalFilter = NULL;
-    thePassAllFilter = NULL;
+    theSysLogEventLogger = nullptr;
+    theManager = nullptr;
```

Comment:
I think you meant to remove this line.

Reply:
hm yes, 1st version used theIManager for logging, so didn't notice during testing, but should have been removed anyway.